client/index.html

```diff
@@ -221,14 +218,14 @@ <h4>Beware</h4>
       </ul>
 
       <details>
-        <summary><h4>Advanced</h4></summary>
+        <summary>
+          <h4>Advanced</h4>
+        </summary>
         <ul>
-          <li><a class="QueryLink"><code>last 2 major versions</code></a> all
-            minor
-            and patch releases of the last 2 major versions.
+          <li><a class="QueryLink"><code>last 2 major versions</code></a>
+            all minor and patch releases of the last 2 major versions.
           </li>
-          <li><a class="QueryLink"><code>last 2 Safari major
```

Comment:
This mid-anchor line break was the root cause of the problem (demonstrated below)
<img width="894" alt="image" src="https://user-images.githubusercontent.com/2292782/232931410-b61622a5-5f1e-4dbc-9560-e4d8b41df9b2.png">

Reply:
Should we change the query parsing JS instead? https://github.com/browserslist/browsersl.ist/blob/main/client/view/QueryLink/QueryLink.js

It will be safer solution. Current PR doesn’t protect us from future bugs on HTML edits.

Reply:
> It will be safer solution. Current PR doesn’t protect us from future bugs on HTML edits.

\+
We should be able to work with any HTML formatting. [Made a PR](https://github.com/browserslist/browsersl.ist/pull/496) with removing `\n` and duplicate spaces in QueryLink.js